discord/user.go

```diff
@@ -25,6 +25,9 @@ type User struct {
 
 	Locale string `json:"locale,omitempty"`
 	Email  string `json:"email,omitempty"`
+
+	Banner      Hash  `json:"banner,omitempty"`
+	AccentColor Color `json:"accent_color,omitempty"`
```

Comment:
On second thought, this field could probably be called `Accent`, since it's
shorter but is just as descriptive with the type name.

Reply:
:+1: Updated